CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Contributing to fullstory-babel-plugin-annotate-react
+
+Thanks for taking the time to contribute!
+
+Here are some important resources to consider:
+
+  * [FullStory's Annotate React plugin for Web & Native](https://help.fullstory.com/hc/en-us/articles/360049493054-FullStory-s-Annotate-React-plugin-for-Web-Native)
+  * [Getting Started with FullStory React Native Capture](https://help.fullstory.com/hc/en-us/articles/360052419133)
+  * [README.md](./README.md)
```

Comment:
Nit: We could probably drop the file extension in the link text
```suggestion
  * [README](./README.md)
```

Reply:
fixed! and moved this to the top.